scripts/lib/aws.sh

```diff
@@ -21,41 +21,21 @@ DEFAULT_AWS_CLI_VERSION="2.0.52"
 # To use a specific version of the AWS CLI, set the ACK_AWS_CLI_IMAGE_VERSION
 # environment variable, otherwise the value of DEFAULT_AWS_CLI_VERSION is used.
 daws() {
-    aws_cli_profile_env=$([ ! -z "$AWS_PROFILE" ] && echo "--env AWS_PROFILE=$AWS_PROFILE")
+    aws_cli_profile_env=$([ -n "$AWS_PROFILE" ] && echo "--env AWS_PROFILE=$AWS_PROFILE")
     aws_cli_img_version=${ACK_AWS_CLI_IMAGE_VERSION:-$DEFAULT_AWS_CLI_VERSION}
     aws_cli_img="amazon/aws-cli:$aws_cli_img_version"
-    docker run --rm -v ~/.aws:/root/.aws:z $(echo $aws_cli_profile_env) -v $(pwd):/aws "$aws_cli_img" "$@"
+    docker run --rm -v ~/.aws:/root/.aws:z "${aws_cli_profile_env[@]}" -v "$(pwd)":/aws "$aws_cli_img" "$@"
```

Comment:
is `aws_cli_profile_env` an array?

Reply:
Ah good catch - it's not. It's a string with a CLI arg. I might just rewrite this function a bit to make it clearer